apps/core/lib/core/schema/provider_scaffold.ex

```diff
@@ -0,0 +1,20 @@
+defmodule Core.Schema.ProviderScaffold do
+  defstruct [:name, :content]
+  @providers ~w(aws gcp azure equinix)a
+
+  def available() do
```

Comment:
small style nit, but you can make these one-line functions a bit more concise with:

```elixir
def available(), do: @available
```